common/source/docs/common-telemetry-rockblock.rst

```diff
@@ -28,36 +28,47 @@ of manual control (outside of low-rate GUIDED mode "fly to" locations) is not po
 The RockBLOCK has a latency of 20-60 seconds and is only able to send 1 message per 30 second period (or greater). This period is
 configurable via the ``RCK_PERIOD`` parameter.
 
+If desired, the RockBlock can automatically activate on loss-of-GCS-telemetry. This is done by setting the ``RCK_FORCEHL`` parameter to 2.
+The RockBLOCK will then automatically activate when telemetry is lost for ``RCK_TIMEOUT`` seconds. When telemetry is restored,
+the RockBLOCK will deactivate.
+
 .. note:: Any modem using the Iridium 9602 or 9603 chipset *should* work, though only the RockBLOCK 9603N modem has been tested.
 
-Supported MAVLink command messages
-==================================
+Supported MAVLink messages
+==========================
 
-- CMD_NAV_RETURN_TO_LAUNCH
-- CMD_NAV_LAND
-- CMD_NAV_TAKEOFF
-- CMD_NAV_VTOL_TAKEOFF
-- CMD_NAV_VTOL_LAND
-- CMD_DO_SET_MODE
-- CMD_MISSION_START
-- CMD_COMPONENT_ARM_DISARM
-- CMD_CONTROL_HIGH_LATENCY
+The following MAVLink messages are supported from the GCS to the vehicle:
+- COMMAND_LONG
+- COMMAND_INT
+- MISSION_ITEM_INT
+- MISSION_SET_CURRENT
 
-Setup
-=====
+Withing the COMMAND_LONG and COMMAND_INT messages, the following commands are supported:
+- MAV_CMD_NAV_RETURN_TO_LAUNCH
+- MAV_CMD_NAV_LAND
+- MAV_CMD_NAV_TAKEOFF
+- MAV_CMD_NAV_VTOL_TAKEOFF
+- MAV_CMD_NAV_VTOL_LAND
+- MAV_CMD_DO_SET_MODE
+- MAV_CMD_DO_CHANGE_SPEED
+- MAV_CMD_DO_SET_SERVO
+- MAV_CMD_DO_PARACHUTE
+- MAV_CMD_MISSION_START
+- MAV_CMD_COMPONENT_ARM_DISARM
+- MAV_CMD_DO_REPOSITION
+- MAV_CMD_CONTROL_HIGH_LATENCY
+
+
+Ground Setup
+============
 
-Due to limitations of the RockBLOCK web service, all packets in the Vehicle to GCS direction need to go via a public-facing web service.
-See `here <https://docs.rockblock.rock7.com/docs/integration-with-application>`__ for details.
-For the purposes of this configuration, the free `adafruit.io <https://io.adafruit.com/>`__ service is used.
+The RockBLOCK messages are sent and received via the Rock7 gateway. It has an API that allows for
+sending and receiving messages via a web service.
 
-#.  Connect the RockBLOCK modem to a spare UART on the flight controller. Only the +5V, RX, TX and GND lines need to be connected
```

Comment:
I think the connection info and link to lua applet, etc. is still required

Reply:
Yep, re-added.